docs/Custom_Pagers.md

```diff
@@ -26,6 +26,8 @@ git:
 
 ![](https://i.imgur.com/QJpQkF3.png)
 
+A cool feature of delta is --hyperlinks, which renders clickable links for the line numbers in the left margin, and lazygit supports these. To use them, set the `pager:` config to `delta --dark --paging=never --line-numbers --hyperlinks`. If you are a VS Code user, I recommend to add `--hyperlinks-file-link-format="vscode-insiders://file/{path}:{line}`; this allows you to click on an underlined line number in the diff to jump right to that same line in your editor.
```

Comment:
Are you using `vscode-insiders` because you personally use that? Or does that work for any vscode user? If it's the former we should assume the users aren't on the insiders edition.

Reply:
Ah yes, I was meaning to remove the `-insiders` but forgot. Fixed in 60d3b6a9ba41.